packages/mariadb/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "bugs": "https://github.com/sequelize/sequelize/issues",
+  "description": "MariaDb Connector for Sequelize",
+  "exports": {
+    ".": {
+      "import": {
+        "types": "./lib/index.d.mts",
+        "default": "./lib/index.mjs"
+      },
+      "require": {
+        "types": "./lib/index.d.ts",
+        "default": "./lib/index.js"
+      }
+    }
+  },
+  "main": "./lib/index.js",
+  "types": "./lib/index.d.ts",
+  "sideEffects": false,
+  "homepage": "https://sequelize.org",
+  "license": "MIT",
+  "name": "@sequelize/mariadb",
+  "repository": "https://github.com/sequelize/sequelize",
+  "scripts": {
+    "build": "../../build-packages.mjs mariadb",
+    "test": "concurrently \"npm:test-*\"",
+    "test-typings": "tsc --noEmit --project tsconfig.json",
+    "test-exports": "../../dev/sync-exports.mjs ./src --check-outdated",
+    "sync-exports": "../../dev/sync-exports.mjs ./src"
+  },
+  "type": "commonjs",
+  "version": "0.0.0-development",
+  "publishConfig": {
+    "access": "public"
+  },
+  "dependencies": {
+    "@sequelize/core": "workspace:*",
+    "@sequelize/utils": "workspace:*",
+    "dayjs": "^1.11.5",
```

Comment:
Prod dependency, shouldn't be pinned :)

Reply:
Done now.

Reply:
Unfortunately that was an accidental request I think, unless we have a reason to pin dayjs specifically that I didn't know about ^^'

Reply:
I've reverted this now.

Reply:
They should indeed not be pinned as prod dependency, I just remarked it because I was different from the others. Not sure why Renovate is not picking that up. 
I'll do a manual PR to unpin the prod dependencies in the various packages

Reply:
Some packages are pinned due to having published a version we're not compatible with